### PR TITLE
Update & localization

### PR DESCRIPTION
**Pull request by javatlacati**
Hello @junichi11 I'd like to merge the changes with your original try.
I've found the original author that is on github too ( And I hope can to be amused with this work ).
In adition I'm now in the netbeans localization project at java.net so I hope we can benefit from the netbeans developer team.
Cheers! :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # NetBeans Encoding Plugin
 This plugin opens/saves a file with a specified encoding.
+Este plugin abre/guarda un archivo con una codificación especificada
```
Comment (junichi11) [inline README.md]: Sorry, please don't add translations to this file.

File: README.md [hunk 2]
```diff
@@ -15,6 +16,11 @@ Current file encoding will be shown on Toolbar. If you want to change encoding,
 
 If you don't want to show it, uncheck `View > Toolbar > Encoding`.
 
+La codificación actual se mostrará en la Barra de Herramientas. Si desea cambiar la codificación, por favor seleccione la codificación específica.
+( Nota:Entonces, el archivo se cierra y se vuelve a abrir.)
+
+Si no desea que se muestre desseleccione la casilla `View > Toolbar > Encoding`.
+
```
Comment (junichi11) [inline README.md]: The same as the above.

File: README.md [hunk 3]
```diff
@@ -22,3 +28,5 @@ If you don't want to show it, uncheck `View > Toolbar > Encoding`.
 Cloned from encoding module of main/contrib repository. (Tomas Zezula)
 
 Also see https://netbeans.org/bugzilla/show_bug.cgi?id=114123
+
+Contacto: scherzo16@gmail.com
```
Comment (junichi11) [inline README.md]: Sorry, please don't add contact because a user may be confused.

File: README_es.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## License
+[Common Development and Distribution License (CDDL) v1.0 and GNU General Public License (GPL) v2](http://netbeans.org/cddl-gplv2.html)
+
+ ----
+Cloned from encoding module of main/contrib repository. (Tomas Zezula)
+Also see https://netbeans.org/bugzilla/show_bug.cgi?id=114123
+
+Translator : @javatlacati
+Contact: scherzo16@gmail.com
```
Comment (junichi11) [inline README_es.md]: Could you please change the following?

Translator : @javatlacati (scherzo16@gmail.com)

File: src/com/junichi11/netbeans/modules/encoding/Bundle.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
 OpenIDE-Module-Display-Category=Utilities
 OpenIDE-Module-Long-Description=\
     This modules provides actions to open or save a file in a user given encoding. \
-    This actions can be used o convert file to different encodings.
+    This actions can be used on convert file to different encodings.
```
Comment (junichi11) [inline src/com/junichi11/netbeans/modules/encoding/Bundle.properties]: :+1:

**Comment by junichi11** [comment]
Hello @javatlacati,

Thanks a lot for your PR :smile: 
I left a few comments. Just have a look at them ;)

**Comment by junichi11** [comment]
@javatlacati Could you fix the above?

**Comment by javatlacati** [comment]
Sure... Just been busy :blush: 

Anyway.. if you want to help me modifying it I won't be angry :smile:

**Comment by javatlacati** [comment]
Modified. Please feel free to make any observation. ( When I started this project haven't heard of markdown. I think I can do something about now )

**Comment by junichi11** [comment]
@javatlacati Thanks a lot! I've merged them.

**Pull request closed without merging**